python/cuml/test/test_random_forest.py

```diff
@@ -153,58 +182,9 @@ def test_rf_regression(datatype, split_algo, mode, column_info,
 
 
 @pytest.mark.parametrize('datatype', [np.float32])
-@pytest.mark.parametrize('column_info', [unit_param([20, 10]),
-                         quality_param([200, 100]),
-                         stress_param([500, 350])])
-@pytest.mark.parametrize('nrows', [unit_param(500), quality_param(5000),
-                         stress_param(500000)])
-def test_rf_classification_default(datatype, column_info, nrows):
-
-    ncols, n_info = column_info
-    X, y = make_classification(n_samples=nrows, n_features=ncols,
-                               n_clusters_per_class=1, n_informative=n_info,
-                               random_state=0, n_classes=2)
-    X = X.astype(datatype)
-    y = y.astype(np.int32)
-    X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=0.8,
-                                                        random_state=0)
-    # Initialize, fit and predict using cuML's
-    # random forest classification model
-    cuml_model = curfc()
-    cuml_model.fit(X_train, y_train)
-    fil_preds = cuml_model.predict(X_test, predict_model="GPU")
-    cu_preds = cuml_model.predict(X_test, predict_model="CPU")
-    fil_preds = np.reshape(fil_preds, np.shape(cu_preds))
-
-    fil_acc = accuracy_score(y_test, fil_preds)
-    cu_acc = accuracy_score(y_test, cu_preds)
-
-    score_acc = cuml_model.score(X_test, y_test)
-    assert cu_acc == pytest.approx(score_acc)
-
-    # sklearn random forest classification model
-    # initialization, fit and predict
-    if nrows < 500000:
-        sk_model = skrfc(max_depth=16, random_state=10)
-        sk_model.fit(X_train, y_train)
-        sk_preds = sk_model.predict(X_test)
-        sk_acc = accuracy_score(y_test, sk_preds)
-        assert fil_acc >= (sk_acc - 0.07)
-    assert fil_acc >= (cu_acc - 0.02)
-
-
-@pytest.mark.parametrize('datatype', [np.float32])
-@pytest.mark.parametrize('column_info', [unit_param([20, 10]),
-                         quality_param([200, 100]),
-                         stress_param([500, 350])])
-@pytest.mark.parametrize('nrows', [unit_param(2000), quality_param(25000),
-                         stress_param(500000)])
-def test_rf_regression_default(datatype, column_info, nrows):
+def test_rf_regression_default(large_dataset, datatype):
```

Comment:
Please correct me if i am missing something but, it seems like the `large_dataset` fixture use the `make_classification` function and the code is using it for regression tests?